lib/OpenQA/WebAPI/Controller/API/V1.pm

```diff
@@ -66,11 +66,12 @@ sub auth {
         }
     }
 
-    if ($user) {
+    if ($user && ($self->is_operator($user) || $self->is_admin($user))) {
         $self->app->log->debug(sprintf "API auth by user: %s, operator: %d", $user->username, $user->is_operator);
         $self->stash(current_user => {user => $user});
-        return $self->is_operator($user);
+        return 1;
```

Comment:
2 reasons: it was forbidden even before (is_operator return 0 thus Mojo stop processing and connection is then dropped by apache after timeout) and because even calls like asset delete, template modifications etc. are protected only by this.